### ISSUE TITLE
Widen the scope from emoji input to more general Unicode character input ?

### ISSUE THREAD

**Issue by odoepner** [rugk/awesome-emoji-picker#124]
## Background
I love this extension and use it all the time on Firefox.
I would like to use the same approach for entering other non-ASCII characters.
For example, German umlauts, French letters, Scandinavian letters, etc.
Sub-string search in English standard Unicode character descriptions would be great.

## Proposed solution
Maybe there could be a checkbox to enable the extended (Unicode) behaviour ?
By default the Emoji search could work just as before.
Maybe my idea is more suitable to a clone and modify development model (i.e. separate extension)?

## Alternatives
Alternatives I use, are bookmarked Unicode pages, with CTRL+F Search in page and manual mark and copy.

## Additional context
I use Firefox on Debian GNU/Linux where Unicode character entry is limited,
Memorizing Unicode hex codes and using CTRL+Shift+u in GTK applications is hard and cumbersome.

**Comment by rugk** [comment]
Hmm interesting idea, however I wonder whether it would make the add-on nearly unusable. There are so many of these symbols and how should we filter it? Compare it with your os or writer programs character selection screen and you see these are heck a lot of symbols.

Furthermore, how can you search for it? Okay maybe Unicode has some descriptions but these are often hardly useful, even with emojis. (cf emojipedia for these). That's why the current dataset provided [by emoji-mart](https://github.com/missive/emoji-mart) uses additional search strings.

And technically it could get complicated as we may be able to define these as custom emojis technically in emoji-mart, but they are then always at the beginning etc.

A whole fork may not be easier, given the selection process cannot he easily taken from the memtioned lib, but would need to be implemented independently.

So basically I guess this could only be an idea for a whole new add-on with a separate selection mechanism. That'd a lot of work.

For now, I'd could suggest you to use/switch between different keyboard layouts in your is that have these keys, or some third-party program/selection window as said.

**Comment by odoepner** [comment]
I guess I was dreaming of a Firefox extension that does what this website does: https://unicode-table.com/en/

**Comment by rugk** [comment]
Oh and yeah, as you can see by that website, it's huge and it is quite complex/not really searchable. So this seems to support what I've previously explained about the challenges/problems that would be involved here.

Thus, i unfortunately have to close this issue as it is unrealistic to be solved/done/added (in this add-on at least). Thanks for the suggestion in any way though.
If you have further questions, feel free to open a new issue or… if on the same topic… comment here…

**Comment by odoepner** [comment]
Ok, thanks for the conversation.
